.gitignore

```diff
@@ -1,8 +1,2 @@
-pkg
-bin
-hound.sublime-workspace
-data
 .vagrant
-pub/index.html
-pub/excluded_files.html
-config.json
```

Comment:
We probably want to retain this line, right? For people who are hacking on Hound. bin and *.sublime-workspace should also stay I would think. 

We also probably want to ignore ui/bindata.go